Debounce/Generic/Debouncer.cs

```diff
@@ -367,6 +377,36 @@ public TimeSpan TimingGranularity
         get => GetField(ref _TimingGranularity);
         set => SetField(ref _TimingGranularity, value, false);
     }
+
+    long _DebounceAfterTriggerCount = long.MaxValue;
```

Comment:
I think this must be `int.MaxValue`, given that Lists lengths are `int`.

Reply:
Right, done.

Reply:
In a later commit (I've got a monster refactor prepared...) we will add a comment that the regular debouncer (due to the lockless trigger) may not honor this setting perfectly...